ide/git/nbproject/project.properties

```diff
@@ -29,8 +29,4 @@ test.config.stable.includes=**/*Test.class
 test.config.stableBTD.includes=**/*Test.class
 test.config.stableBTD.excludes=\
     **/ConnectTest.class,\
-    **/DiffTest.class,\
-    **/ExternalChangesTest.class,\
-    **/FilesystemInterceptorTest.class,\
-    **/getTopmostTest.class,\
-    **/MergeTest.class
+    **/ExternalChangesTest.class
```

Comment:
i checked and `ide/git` tests aren't run atm.

only found `ide/libs.git` and `ide/versioning*`. Since you fixed a lot of tests int he second commit, would you mind adding them near the `ide/libs.git` step to the workflow?

I will move all versioning and git tests to the versioning job the next time I update the yaml, but for now lets keep them in the IDE job near `libs.git`.

Reply:
also: consider making this `test.config.default.excludes=..`

so that only the working tests are run by default.